### ISSUE TITLE
When inserting a Text control in the FloatingActionButton content, bgcolor has no action.

### ISSUE THREAD

**Issue by nascin** [flet-dev/flet#196]
When inserting a Text control in the FloatingActionButton content, bgcolor has no action.

code example:

```
import flet
from flet import Column, FloatingActionButton, Page, Text

def main(page: Page):
    page.add(
        Column(
            [
                FloatingActionButton(
                    bgcolor='#ed90e1',
                    content=Text("I'm Big!", size=50), width=300, height=200
                ),
            ]
        )
    )
    page.update()

flet.app(target=main)
```

**Comment by FeodorFitsner** [comment]
Confirmed, it's a bug. Will publish a fix shortly.

**Comment by nascin** [comment]
I have now verified that the same bug occurs with icon.

**Comment by FeodorFitsner** [comment]
> I have now verified that the same bug occurs with icon.

Drop me a repro sample please?

**Comment by nascin** [comment]
Code example:
```
import flet
from flet import Column, FloatingActionButton, Page, Text, icons

def main(page: Page):
    page.add(
        Column(
            [
                FloatingActionButton(
                    icon=icons.CHECK,
                    content=Text("I'm Big!", size=50), width=300, height=200
                ),
            ]
        )
    )
    page.update()

flet.app(target=main)
```

**Comment by FeodorFitsner** [comment]
That's because `content` overrides everything. It should be either `icon` or/and `text` or `content`.

**Comment by nascin** [comment]
Now I get it. Thanks!

**Comment by FeodorFitsner** [comment]
Should be fixed in just released Flet 0.1.51.
